requirements.txt

```diff
@@ -1,2 +1,4 @@
-tf-nightly-2.0-preview
+tf-nightly
```

Comment:
I thought we are going to develop on tf-nightly-2.0-preview?

Reply:
We can't, tfp-nightly and tf2 don't work together at the moment